apis/duck/v1alpha1/addressable_types.go

```diff
@@ -26,17 +26,18 @@ import (
 
 // Addressable provides a generic mechanism for a custom resource
 // definition to indicate a destination for message delivery.
-// (Currently, only hostname, port and path are supported, and HTTP is implied. In the
-// future, additional schemes may be supported
-// ala UI may also be supported.)
-
+// Hostname is deprecated as of v0.6 and replaced with URL. It will be removed in ~v0.7
+// URL string currently supports scheme(http or https), authority(host and optional port), and path
+// Example: http://myhost:8080/ResourceGroup/ResourceName (http://<authority>/<path>)
 // Addressable is the schema for the destination information. This is
 // typically stored in the object's `status`, as this information may
 // be generated by the controller.
 type Addressable struct {
+	// Hostname is deprecated as of v0.6 and replaced with URL. It will be removed in ~v0.7
```

Comment:
Add trailing punctuation.

Reply:
Change `Hostname` to `DeprecatedHostName` and add a comment that starts with: 

```
// Deprecated: yada yada yada
```